packages/unmagic/unmagic.1.0.4/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+version: "1.0.4"
+maintainer: "jun.furuse@gmail.com"
+authors: "Jun Furuse"
+homepage: "https://bitbucket.org/camlspotter/unmagic/"
+bug-reports:
+  "https://bitbucket.org/camlspotter/unmagic/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/unmagic"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "jbuilder" {build}
+  "spotlib"
+  "typerep" {>= "v0.10.0"}
+  "ppx_typerep_conv" {build}
+  "ppx_deriving"
```

Comment:
Are you sure that you need ppx_deriving, if you're using `ppx_typerep_conv` you're using `ppx_type_conv` I'm assuming? Which means that you should only depend on `ppx_typerep_conv` here.